security-admin/src/main/java/org/apache/ranger/security/handler/RangerAuthenticationProvider.java

```diff
@@ -94,79 +94,79 @@ public Authentication authenticate(Authentication authentication)
 				}
 			}
 		} else {
-		String sha256PasswordUpdateDisable = PropertiesUtil.getProperty("ranger.sha256Password.update.disable", "false");
-		if (rangerAuthenticationMethod==null) {
-			rangerAuthenticationMethod="NONE";
-		}
-		if (authentication != null && rangerAuthenticationMethod != null) {
-			if (rangerAuthenticationMethod.equalsIgnoreCase("LDAP")) {
-				authentication = getLdapAuthentication(authentication);
-				if (authentication!=null && authentication.isAuthenticated()) {
-					return authentication;
-				} else {
-					authentication=getLdapBindAuthentication(authentication);
+			String sha256PasswordUpdateDisable = PropertiesUtil.getProperty("ranger.sha256Password.update.disable", "false");
```

Comment:
is this just indentation fix? If it is, I would remove it from this PR, because patch is cleaner without it. I would contribute different patch first to fix the indentation, or let others do it. General hint - keep style changes as separate PRs from actual code changes. Much easier to review.